### PR TITLE
Fix some warnings

### PR DESCRIPTION
**Pull request by mhmdanas**
#### What is it?
- [x] Codebase improvement (dev facing)

#### Description of the changes in your PR
Fix a number of warnings.

#### APK testing 

On the website the APK can be found by going to the "Checks" tab below the title and then on "artifacts" on the right.

#### Due diligence
- [x] I read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md).

### CODE REVIEW

File: app/src/main/java/org/schabi/newpipe/local/subscription/dialog/FeedGroupDialog.kt [hunk 1]
```diff
@@ -127,7 +128,7 @@ class FeedGroupDialog : DialogFragment(), BackPressable {
 
         if (android.os.Build.VERSION.SDK_INT < android.os.Build.VERSION_CODES.LOLLIPOP) {
             // KitKat doesn't apply container's theme to <include> content
-            val contrastColor = ColorStateList.valueOf(resources.getColor(R.color.contrastColor))
+            val contrastColor = ColorStateList.valueOf(ContextCompat.getColor(requireContext(), R.color.contrastColor))
```
Comment (mhmdanas) [inline app/src/main/java/org/schabi/newpipe/local/subscription/dialog/FeedGroupDialog.kt:131]: I don't have a KitKat device, so I unfortunately couldn't test this.

File: app/src/main/java/org/schabi/newpipe/local/subscription/SubscriptionFragment.kt [hunk 2]
```diff
@@ -207,7 +207,7 @@ class SubscriptionFragment : BaseStateFragment<SubscriptionState>() {
         super.onActivityResult(requestCode, resultCode, data)
         if (data != null && data.data != null && resultCode == Activity.RESULT_OK) {
             if (requestCode == REQUEST_EXPORT_CODE) {
-                val exportFile = Utils.getFileForUri(data.data!!)
+                val exportFile = Utils.getFileForUri(data.data!!)!!
```
Comment (TiA4f8R) [inline app/src/main/java/org/schabi/newpipe/local/subscription/SubscriptionFragment.kt:210]: It seems uneeded:

```
> Task :app:compileDebugKotlin
[...]
w: /home/runner/work/NewPipe/NewPipe/app/src/main/java/org/schabi/newpipe/local/subscription/SubscriptionFragment.kt: (210, 66): Unnecessary non-null assertion (!!) on a non-null receiver of type File
```

**Comment by TiA4f8R** [comment]
Also, can you fix this?

```
> Task :app:javaPreCompileDebug
warn: removing resource org.schabi.newpipe.debug.fixsomewarnings:string/popup_resizing_indicator_title without required default value.
warn: removing resource org.schabi.newpipe.debug.fixsomewarnings:string/related_streams_tab_description without required default value.
```

It seems that some translations of these two removed strings haven't been removed.

File: app/src/main/java/org/schabi/newpipe/local/subscription/SubscriptionFragment.kt (hunk 2 above)
Comment (mhmdanas) [reply app/src/main/java/org/schabi/newpipe/local/subscription/SubscriptionFragment.kt:210]: Oh, I misunderstood the error. Seems like `parentFile` is the one that is nullable, not `exportFile`. _facepalm_

**Comment by mhmdanas** [comment]
Thanks for the pointer; I had wanted to fix those errors, but I didn't really know what they meant. I'll fix them.

**Review by TiA4f8R — APPROVED** [review]
Seems good, now we need someone who can test the changes on a KitKat device.

**Comment by mhmdanas** [comment]
@mqus @MD77MD mind testing the feed group dialog on KitKat? Preferably with multiple themes.

**Comment by mqus** [comment]
@mhmdanas Could you be more specific? What workflow(s) should work and what should we look out for?

**Comment by mhmdanas** [comment]
@mqus sorry, I'm not too sure what the code does, so I can't be much more specific, but I'll try.

Basically, I just changed the way a color is being loaded, so all you have to do open the subscriptions tab, press the "new" button in the "channel groups" section, and then press "Select subscriptions"; after that just please check that the colors are correct (e.g. press the search button), preferably in multiple themes.

**Comment by mqus** [comment]
I just notice that the UI reloads after changing a theme(normal) and reloads again when I've closed the settings(seems unnecessary), even though the scheme was already applied. But this only occurred sometimes, I can't reproduce it right now. The colors are fine :+1:

**Comment by mhmdanas** [comment]
@TiA4f8R now that this was confirmed to work fine on KitKat, can you please go ahead and merge this?

**Pull request merged**